controllers/rhmi/rhmi_controller.go

```diff
@@ -817,7 +821,7 @@ func (r *RHMIReconciler) handleUninstall(installation *rhmiv1alpha1.RHMI, instal
 			if err != nil {
 				merr.Add(err)
 			}
-			return retryRequeue, nil
+			return nil
```

Comment:
if we did return a Requeue - it means to me that at this point, the current controller loop should stop, and it should be requeued. By returning nil here we are saying "doesn't matter, just continue" - if we plan on changing logic like so, please ensure extended verification steps are in place to be sure it's working as expected.

Reply:
It'll not skip it and continue - it will return nil and be requeued as on the line 387 the controller will be requeued if `handleUninstall` returns nil: https://github.com/integr8ly/integreatly-operator/blob/edbe2c72475b1b74ad724685fa540aabc301d366/controllers/rhmi/rhmi_controller.go#L383-L387